bazel_tools/haskell-ghc-lib.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/compiler/deSugar/DsMonad.hs b/compiler/deSugar/DsMonad.hs
```

Comment:
I suppose I'm not up-to-date regarding the infrastructure around our patched GHC, but do we need to check this patch in here too?

Reply:
Similar to your other comment: Now that the PR to our GHC fork has been approved, I’ll remove that cruft. This was just temporary for testing it out.

Reply:
That makes sense. Thanks for the explanation.

Reply:
Merged the PR and switched over.